webapp/app.py

```diff
@@ -849,7 +977,8 @@ def setup_apispec(handlers):
     })
     # Register public API handlers to apispec
     for handler in handlers:
-        if handler[0].startswith(r"/api/v1/"):
+        if handler[0].startswith(r"/api/v1/") \
```

Comment:
We could write this with r"/api/v[12]/" instead of two compares, yeah?

Reply:
I tried doing that, but it didn't work.  Spent an hour trying to figure out what the correct regex is.  Gave up and did this.  :(